big.go

```diff
@@ -5,7 +5,10 @@
 
 package prnm
 
-import "math/big"
+import (
+	"errors"
```

Comment:
You're now mixing `errors` and `pkg/errors`. I think we should stick to `pkg/errors`, since it's used in the framework already anyways.